.ci/pytorch/win-test-helpers/setup_pytorch_env.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Can you please add a comment here and in the corresponding .bat file to copy any changes over to the matching file?

On the off chance someone tried to modify these files before your complete/revert this migration, lets leave some breadcrumb to help prevent the files from diverging (feeble as it may be)